plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerStartJobResult.kt

```diff
@@ -7,6 +7,7 @@ sealed class CodeModernizerStartJobResult {
     data class ZipCreationFailed(val reason: String) : CodeModernizerStartJobResult()
     data class Started(val jobId: JobId) : CodeModernizerStartJobResult()
     data class UnableToStartJob(val exception: String) : CodeModernizerStartJobResult()
+    data class UnabletoStartOrUploadJob(val url: String) : CodeModernizerStartJobResult()
```

Comment:
Change this to `UnabletoUploadJob` and we can keep `UnableToStartJob` a separate error.